src/domain/journey/space/pages/SpaceSettings/SpaceSettingsView.tsx

```diff
@@ -284,6 +330,25 @@ export const SpaceSettingsView: FC<SpaceSettingsViewProps> = ({ journeyId, journ
               }}
             />
           </PageContentBlock>
+
+          <PageContentBlock sx={{ borderColor: theme.palette.error.main }}>
```

Comment:
There should be a check only for GAs and Creators to delete a Space. Not every Admin should be able to delete (see the option) space.